content/for-delivery-engineers/tutorials/04-preparing-pac-env/adding-github-token.md

```diff
@@ -16,7 +16,7 @@ You can check secrets documentation to read more on these secrets.
 ## Prerequisites
 
 * Infra GitOps Repository is configured.
-* Delivery Engineer added as the owner of root-tenant. The root tenant makes sure that the delivery engineer is able to login to Vault with OIDC and is able to view the ArgoCD application created for bootstrapping Infra repository.
+* Delivery Engineer added as the owner of root-tenant.
```

Comment:
@AsfaMumtaz why is root-tenant required? we need to explain that as well; what if it is not created?

Reply:
It is at the bottom in a note